engine/src/main/scala/cromwell/webservice/routes/wes/WesState.scala

```diff
@@ -0,0 +1,54 @@
+package cromwell.webservice.routes.wes
+
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
+import cromwell.core._
+import spray.json.{DefaultJsonProtocol, JsString, JsValue, RootJsonFormat}
+
+object WesState {
+  sealed trait WesState extends Product with Serializable
+  case object UNKNOWN extends WesState
```

Comment:
IF THERE'S ANYTHING THAT COULD BE DONE ABOUT THESE IDENTIFIER NAMES THAT WOULD REALLY BE GREAT

Reply:
Considering I historically have always shot down the argument I was about to make, I suppose I shall change these :)

Reply:
However, the flip side is that all these capitalizations shall reappear in the `write` method of the JSONifier below but 🤷‍♂️